src/config.ts

```diff
@@ -2,7 +2,10 @@ import * as path from 'path';
 import * as fs from 'fs';
 
 export const EXPORTED_PATH = path.join(__dirname, '../exported');
-
+if(!fs.existsSync(EXPORTED_PATH)) {
+  console.log(`${EXPORTED_PATH} don't exist, creating`);
+  fs.mkdirSync(EXPORTED_PATH);
```

Comment:
what about exceptions becaufe of file system rights?

Reply:
Actually, we can't manage this exception anyhow
So, maybe it's better to just let it fall?

Reply:
ok